Google Maps

kjrcda opened the pull request:
Working map up and running. Needs functionality.

kjrcda committed:
Map functioning

Got a map route and map implemented and running on page load.

paymog commented on webapp/views/partials/map.jade:
```diff
@@ -0,0 +1,8 @@
+    h1 Pick a Place on the Map
+
+    //this should actually be "div(ui-view)" but for some reason that doesn't work...
+    //See http://goo.gl/TVBKXO
```
These comments should be removed since you removed the `ui-view` tag.

paymog commented:
`bower.json` hasn't been updated to include `lodash` or the `angularmaps......` packages.

paymog commented on webapp/views/layout.jade:
```diff
@@ -13,6 +13,9 @@ html(lang='en', ng-app='lunchr')
         script(src='bower_components/angular-ui-router/release/angular-ui-router.js')
         script(src='javascripts/angular-app.js')
         script(src='javascripts/controllers.js')
+        script(src='bower_components/lodash/dist/lodash.js')
```
why do we need this?

kjrcda replied:
I am not 100% sure, I can look into if we need it or not. When I installed the framework lodash came with it. I think the map depends on it.

paymog replied:
What happens if you comment this out?

kjrcda replied:
It breaks. So yes we do need it.

kjrcda committed:
Pkg +cmt

- Packages updated to have angular-google-maps. Lodash is a dependency
for that framework, and is installed via the ang-ggl-map's bower.json.
- Removed the old comment

BrandonWade committed:
Added support for geolocation from browser.

Loading map page now automatically prompts user for their location.
If the user accepts, the embedded map will scroll to their current
location.`

paymog commented:
Write tests for the maps controller

kyleleung75 commented:
Heyo, we are going to close this until we have tests for the map stuff and because of the commits brandon made today

kjrcda commented:
Why close it though? we could just keep it open so we dont forget about it?

kyleleung75 commented:
Just so that it is very obvious we aren't going to merge it yet. It can be
reopened can't it?

On Thu, Feb 26, 2015, 5:24 PM kjrcda notifications@github.com wrote:

> Why close it though? we could just keep it open so we dont forget about it?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/paymog/Lunchr/pull/24#issuecomment-76297731.

kjrcda commented:
It can ya. Personally I find the PR help so people can see and review your work.
